kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/DrivingTask.java

```diff
@@ -20,12 +21,12 @@ public String getTaskName() {
 
     @Override
     public void executeTask() {
-        System.out.println(String.format(EXECuTING_TASK_INFO,
+        System.out.println(String.format(EXECUTING_TASK_INFO,
                 this.taskName,
                 this.where,
                 this.using));
 
-        isExexuted = true;
+        isExexuted = false;
```

Comment:
a czemu tak? chyba jak wykonane, to powinno byc jednak true. chodzilo mi o inicjacje w konstruktorze